src/annotator/components/NotebookModal.js

```diff
@@ -0,0 +1,96 @@
+import { createSidebarConfig } from '../config/sidebar';
+import { useEffect, useRef, useState } from 'preact/hooks';
+import classnames from 'classnames/bind';
+
+// FIXME: use the button from the frontend shared package once this is stable.
+import Button from '../../sidebar/components/Button';
+
+/**
+ *
+ * @typedef NotebookIframeProps
+ * @prop {Record<string, any>} config
+ * @prop {string} groupId
+ */
+
+/**
+ * Create the iframe that will load the notebook application.
+ * 
+ * @param {NotebookIframeProps} props
+
+ */
+function NotebookIframe({ config, groupId }) {
+  const notebookConfig = createSidebarConfig(config);
+  // Explicity set the "focused" group
+  notebookConfig.group = groupId;
+  const configParam = encodeURIComponent(JSON.stringify(notebookConfig));
+  const notebookAppSrc = `${config.notebookAppUrl}#config=${configParam}`;
+
+  return (
+    <iframe
+      title={'Hypothesis annotation notebook'}
+      className="Notebook__iframe"
+      // Enable media in annotations to be shown fullscreen
+      allowFullScreen
+      src={notebookAppSrc}
+    />
+  );
+}
+
+/**
+ * @typedef NotebookModalProps
+ * @prop {import('../../types/annotator').EventBus} eventBus
```

Comment:
I commented on this in `types/annotator.js` but it is fine to continue to import the `EventBus` type directly from the `util/emitter` module. The `types/annotator.js` module exists for types used in the annotator which don't have a natural home in a more specific module.